docs/src/pages/components/no-ssr/SimpleNoSsr.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import NoSsr from '@material-ui/core/NoSsr';
+import NoSsr from '@material-ui/unstyled/NoSsr';
```

Comment:
This is not consistent with what we do for the portal that is in the same configuration: https://next.material-ui.com/components/portal/. And inconsistencies open doors. for discussion: why? what's best?

Basically, with Portal, we documents the import from core, and add a dedicated unstyled section: https://next.material-ui.com/components/portal/#unstyled.

Reply:
My rationale was that the components lives in the Unstyled package and is reexported just for compatibility. Having a clear indication that this component can be imported from Unstyled will make the developers use a minimal set of dependencies (i.e. the whole Core is not needed when using code from this demo).

There is a tradeoff, though - some may find it confusing that some components are imported from Core and some from Unstyled. I can make it more clear you can use either option.

What were the arguments for using `@material-ui/core` in Portal's demos?

Reply:
> What were the arguments for using @material-ui/core in Portal's demos?

@michaldudak The idea was to allow developers to find everything under `@material-ui/core` so they don't have to thin about where to import what.

Reply:
Oh actually, how about:

1. We document from core hee
2. Later, we create a isolated docs section for unstyled

Reply:
> What do you mean by "whole"?

Perhaps not the best wording. I meant that it's not necessary to even have Core as a dependency (assuming we also use `Box` from System).

> Oh actually, how about:
> 
> We document from core here for convenience, no extra package to install, to figure out where to import.
> Later, we create an isolated docs section for unstyled, different documentation, for a different audience, with different imports

Sounds good to me.

Reply:
cc @danilo-leal